### ISSUE TITLE
Medievil 2 - Wrong musics playing.

### ISSUE THREAD

**Issue by bslenul** [notaz/pcsx_rearmed#144]
Hey!

Someone on a discord server was complaining about not having the correct musics playing in Medievil 2 with PCSX ReARMed, by curiosity I tried myself and yup, there's something wrong! USA redump version of the game, I haven't tried EUR version but the same person confirmed the bug is also present.

I made the video with RetroArch, but I also tested with the standalone version by cloning this repo on my Linux VM and the results are the same! I also tried every speedhack available and the "SPU IRQ Always Enabled" option as someone else suggested but the issue is still present.

So, here is how it sounds in PCSX ReARMed: https://youtu.be/pZRl6wxTzRU

You can hear that the music completely changes between the 2 rooms.

Now with Beetle PSX: https://youtu.be/mDqKtl_dmqA

Here it's played properly, it's still the same music as it should be!

It's really early ingame but if it helps, here's the state file I made (works fine with both standalone and RetroArch versions of PCSX ReARMed): [MEDIEVIL2-SCUS94564.zip](https://github.com/notaz/pcsx_rearmed/files/3741754/MEDIEVIL2-SCUS94564.zip)

**Comment by RealRaptoReX** [comment]
I was the one that noticed this. I play MediEvil a lot and wanted to play this game again on the PS Classic. 

I first thought it was only the first level but i played till kew gardens. Even in that level, the music plays wrong. If you go inside, it starts to play the music from the freakshow.

I tried everything from different file formats to different dumps of the game, nothing helped, so the issue must be an bug in the playback of the emulator with a missing fix. On youtube, somebody shows off this issue on epsxe 1.7, which was fixed in 2.0.5
